ruby/ext/google/protobuf_c/map.c

```diff
@@ -61,9 +61,13 @@ static void Map_mark(void* _self) {
   rb_gc_mark(self->arena);
 }
 
+static size_t Map_memsize(void* _self) {
+  return sizeof(Map);
```

Comment:
Here I assume that `const upb_Map* map;` is pointing into the `Arena` right?

Reply:
That is correct.